src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/integration/offenderManager/AllocateCaseToOffenderManager.kt

```diff
@@ -375,7 +376,7 @@ class AllocateCaseToOffenderManager : IntegrationTestBase() {
   }
 
   @Test
-  fun `audit data contain required fields`() {
+  fun `audit data contain required fields and only unallocated requirement`() {
```

Comment:
Can we add an assertion here that it does not contain the data for an allocated requirement?